requirements.txt

```diff
@@ -0,0 +1,19 @@
+# build
```

Comment:
Do we truly need this file? I believe we considered adding it in the past, but rejected it because it wasn't clear what versions should go in there and when they should be updated, that it was a bit misleading (this is certainly not the only set of versions that works). If anything, this also looks like something that ought to be generated when creating a release tarball but not tracked in Git...

Reply:
Hmm... Not necessarily.

I was wondering, and just took it from my original setup. The only issue I had with not having it, is the pip install step potentially pulling versions that don't actually work well (or at not compatible anymore, etc)?

Would be happy to have a generated one at build/test time and ingest it in the docker image building step however, so we know it was the same versions as the ones used to run the tests etc!